include/seqan3/alphabet/aminoacid/concept.hpp

```diff
@@ -0,0 +1,133 @@
+ * |    Aspartic acid           | Asp                         | D                           | D                          |
+ * |    Cysteine                | Cys                         | C                           | C                          |
+ * |    Glutamic acid           | Glu                         | E                           | E                          |
+ * |    Glutamine               | Gln                         | Q                           | Q                          |
+ * |    Glycine                 | Gly                         | G                           | G                          |
+ * |    Histidine               | His                         | H                           | H                          |
+ * |    Isoleucine              | Ile                         | I                           | I                          |
+ * |    Leucine                 | leu                         | L                           | L                          |
+ * |    Lysine                  | Lys                         | K                           | K                          |
+ * |    Methionine              | Met                         | M                           | M                          |
+ * |    Phenylalanine           | Phe                         | F                           | F                          |
+ * |    Proline                 | Pro                         | P                           | P                          |
+ * |    Serine                  | Ser                         | S                           | S                          |
+ * |    Threonine               | Thr                         | T                           | T                          |
+ * |    Tryptophan              | Trp                         | W                           | W                          |
+ * |    Tyrosine                | Tyr                         | Y                           | Y                          |
+ * |    Valine                  | Val                         | V                           | V                          |
+ * |    Selenocysteine          | Sec                         | C                           | U                          |
+ * |    Pyrrolysine             | Pyl                         | K                           | O                          |
+ * | Asparagine or aspartic acid| Asx                         | D                           | B                          |
+ * | Glutamine or glutamic acid | Glx                         | E                           | Z                          |
+ * |    Leucine or Isoleucine   | Xle                         | L                           | J                          |
+ * |    Unknown                 | Xaa                         | S                           | X                          |
+ * |    Stop Codon              | N/A                         | W                           | TERMINATOR                 |
+ */
```

Comment:
This table is definitely helpful, but I would move it to the general description of the aminoacid submodule where we have such a table already.

I would merge them and make four columns:

| Full name | Three letter code | One letter code | Remapped in seqan3::aa20

And then write after the table something like:
```
All amino acid alphabets provide static value members (like an enum) for all amino acids in the form of the one-letter representation (except that the stop codon is called ::TERMINATOR). As shown above, alphabets smaller than 27 internally represent multiple amino acids as one.

For most cases it is highly recommended to use seqan3::aa27 as seqan3::aa20 provides no benefits in regard to space consumption (both need 5bits). Use it only when you know you need to interface with other software of formats that only support the canonical set.
```